test/tests/functional/pbs_provisioning_enhancement.py

```diff
@@ -209,14 +225,17 @@ def test_job_wide_provisioining_request(self):
         self.server.expect(JOB, {ATTR_state: 'Q',
                                  ATTR_comment:
                                  (MATCH_RE, 'Not Running: Insufficient ' +
-                                  'amount of resource*')}, id=jid)
+                                  'amount of resource: .*')}, id=jid)
```

Comment:
No need for the .* at the end.  log_match() will do substring matching

Reply:
But this isn't log_match().